src/org/usfirst/frc/team930/robot/subsystems/SwerveDrive.java

```diff
@@ -0,0 +1,201 @@
+package org.usfirst.frc.team930.robot.subsystems;
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class SwerveDrive extends Subsystem {
```

Comment:
Well if you want to pull it out for testing it can't implement any WPILib classes. The Drivetrain subsystem could use this to get the correct values from the output method.